internal/cache/file/cache_handle_test.go

```diff
@@ -820,3 +820,31 @@ func (cht *cacheHandleTest) Test_MultipleReads_CacheHitShouldBeFalseThenTrue() {
 	assert.True(cht.T(), cacheHit)
 	assert.Nil(cht.T(), err)
 }
+
+func (cht *cacheHandleTest) Test_Read_Sequential_Parallel_Download_True() {
```

Comment:
also, add Read_Random test for parallel downloads

Reply:
Added